spec/controllers/works_controller_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe WorksController, type: :controller do
+  context "smoke tests" do
+    context "standard work" do
+      let(:work) { FactoryBot.create(:work, :with_assets)}
+
+      it "shows the work as expected" do
+        get :show, params: { id: work.friendlier_id }, as: :html
+        expect(response.body).to include work.title
+      end
+
+      it 'allows user to download RIS citation' do
+        get :show, params: { id: work.friendlier_id }, as: :ris
+        expect(response.status).to eq(200)
+        expect(response.body).to include "TI  - Test title"
+        expect(response.body).to include "M2  - Courtesy of Science History Institute."
+        expect(response.body).to include "M2  - Courtesy of Science History Institute."
```

Comment:
Duplicate line; will fix in the nex edit.

Reply:
cool, makes sense to test the actual content very limited if at all, since that's already tested in the RIS serializer. 

I guess we could do something to test that it looks like a valid RIS file at all, a bunch of lines matching `/[A-Z0-9]{2}.*\r\n/` or whatever. 

But I'm not worried about it, this is just fine!